### PR TITLE
Added brew formula

### PR DESCRIPTION
**Pull request by kennyg**
This is a follow up for https://github.com/jenkinsci/jenkinsfile-runner/pull/25 that fixes the repo names

### CODE REVIEW

File: Formula/jenkinsfile-runner.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class JenkinsfileRunner < Formula
+    desc "A command line tool to run Jenkinsfile"
```
Comment (michaelneale) [inline Formula/jenkinsfile-runner.rb]: Change it to "Jenkinsfiles" plural I think - reads a bit nice.

**Review by michaelneale — CHANGES_REQUESTED** [review]
Look good - just pluralise Jenkinsfiles I think

File: Formula/jenkinsfile-runner.rb (hunk 1 above)
Comment (kennyg) [reply Formula/jenkinsfile-runner.rb]: Nice catch. I agree.

**Review by michaelneale — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
maybe it is better to use GitHub releases instead of rebuilding the tool locally and requiring Maven from HomeBrew. No objection w.r.t merging though, it can be gradually improved

**Pull request merged**